content/docs/reference/autocert.mdx

```diff
@@ -15,47 +17,42 @@ import TabItem from '@theme/TabItem';
 
-The Autocert setting can be useful in situations where you may not have Pomerium behind a TLS terminating ingress or proxy that is already handling your public certificates on your behalf.
+- Pomerium Zero users should note that Pomerium automatically manages certificates for [starter domains](/docs/concepts/clusters#starter-domain) and [custom domains](/docs/capabilities/custom-domains).
 
-Autocert will incorporate certificates available in the system trust store and certificates set manually in the Pomerium configuration, and they will take precedence over generated certificates when applicable to configured routes.
+:::
+
+## Autocert {#autocert}
 
-Autocert will attempt `HTTP-01`and `TLS-ALPN-01` challenges. It does not support `DNS-01` challenges, required to generate wildcard certificates.
+The **Autocert** setting enables Pomerium to automatically retrieve, manage, and renew public-facing TLS certificates from [Let's Encrypt][letsencrypt] to secure Pomerium-managed route domains and the authenticate service (Autocert is not supported with the [Hosted Authenticate Service](/docs/capabilities/hosted-authenticate-service)).
 
-Kubernetes users should not use Autocert. See the [cert-manager's guide](https://cert-manager.io/docs/tutorials/acme/pomerium-ingress/) for more information.
+Autocert can incorporate also certificates available in the system trust store and certificates set manually in the Pomerium configuration. These certificates will take precedence over generated certificates when applicable to configured routes.
 
-:::warning
+Before Autocert can provision certificates, it must complete either the [HTTP-01](https://letsencrypt.org/docs/challenge-types/#http-01-challenge) or [TLS-ALPN-01](https://letsencrypt.org/docs/challenge-types/#tls-alpn-01) challenge. The HTTP-01 challenge requires internet access to port 80; the TLS-ALPN-01 challenge requires internet access to port 443. Autocert does not support `DNS-01` challenges, which is required to generate wildcard certificates.
 
-By using Autocert, you agree to the [Let's Encrypt Subscriber Agreement](https://letsencrypt.org/documents/LE-SA-v1.2-November-15-2017.pdf). There are [_strict_ usage limits](https://letsencrypt.org/docs/rate-limits/) per domain you should be aware of. Consider testing with [`autocert_use_staging`](/docs/reference/autocert) first.
+You must use the [Autocert Directory](#autocert-directory) setting with Autocert to store and share certificate data between Pomerium services.
 
-Autocert requires that port `443` be accessible from the internet in order to complete a [TLS-ALPN-01 challenge](https://letsencrypt.org/docs/challenge-types/#tls-alpn-01) or port `80` in order to complete an [HTTP-01 challenge](https://letsencrypt.org/docs/challenge-types/#http-01-challenge) (https://letsencrypt.org/docs/challenge-types/#tls-alpn-01).
+:::caution Autocert usage limits
```

Comment:
This part is really specific to Let's Encrypt, not to Autocert in general.

Reply:
Good point. I think since we use LE by default, it still makes sense to include this information somewhere. Maybe we could change the admonition title to "Let's Encrypt usage limits" and change the text say something along the lines of, "If you use the default Let's Encrypt CA with Autocert, you agree to LE's subscriber agreement."